<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>黑体</family>
    <pointsize>15</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="icon">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>9999999</width>
      <height>9999999</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>C:/Users/Administrator/Desktop/IMAGE_iP2SjlZhb_1690890042506.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>400</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>YEFONTJingHongXingKai</family>
      <pointsize>35</pointsize>
     </font>
    </property>
    <property name="text">
     <string>北京题库拓展程序</string>
    </property>
   </widget>
   <widget class="QLabel" name="icon2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>0</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>C:/Users/Administrator/Desktop/out.png</pixmap>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>800</width>
      <height>5</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>YEFONTJingHongXingKai 標準</family>
      <pointsize>1</pointsize>
     </font>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>5</width>
      <height>385</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>535</y>
      <width>800</width>
      <height>5</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>YEFONTJingHongXingKai 標準</family>
      <pointsize>1</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>799</x>
      <y>150</y>
      <width>5</width>
      <height>385</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>411</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="agreement">
       <property name="font">
        <font>
         <family>黑体</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>我已阅读并同意</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_am">
       <property name="font">
        <font>
         <family>黑体</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>用户协议</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>黑体</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>且知晓并认同</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_sm">
       <property name="font">
        <font>
         <family>黑体</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>声明</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>150</y>
      <width>3</width>
      <height>385</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>160</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>YEFONTJingHongXingKai</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>搜索：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="search_keyword">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>160</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>160</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>YEFONTJingHongXingKai</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>数量：</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="search_num">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>160</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ink Free</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="maximum">
     <number>150</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>200</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>YEFONTJingHongXingKai</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>猜你想搜：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="search">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>200</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>YEFONTJingHongXingKai</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>搜索</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hot1">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>205</y>
      <width>35</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>仿宋</family>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string>2022</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hot2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>205</y>
      <width>35</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>仿宋</family>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string>语文数学</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hot4">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>205</y>
      <width>35</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>仿宋</family>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string>2022</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hot5">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>205</y>
      <width>35</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>仿宋</family>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string>月考</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hot3">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>205</y>
      <width>35</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>仿宋</family>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string>英语</string>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>230</y>
      <width>400</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QListWidget" name="search_list">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>250</y>
      <width>360</width>
      <height>180</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ink Free</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="search_choose">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>440</y>
      <width>360</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>YEFONTJingHongXingKai</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pdf_browser">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>460</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>YEFONTJingHongXingKai</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>默认浏览器打开</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pdf_url">
    <property name="geometry">
     <rect>
      <x>585</x>
      <y>460</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>YEFONTJingHongXingKai</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>复制URL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="phone_pdf">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>460</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>YEFONTJingHongXingKai</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>手机扫码下载</string>
    </property>
   </widget>
   <widget class="QPushButton" name="browser_pan">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>500</y>
      <width>160</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>YEFONTJingHongXingKai</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>浏览器跳转百度网盘</string>
    </property>
   </widget>
   <widget class="QPushButton" name="phone_pan">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>500</y>
      <width>180</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>YEFONTJingHongXingKai</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>手机百度网盘扫码获取</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>545</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ink Free</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Version：1.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>545</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ink Free</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Sat Sep  9 2023</string>
    </property>
   </widget>
   <widget class="QPushButton" name="developer">
    <property name="geometry">
     <rect>
      <x>525</x>
      <y>545</y>
      <width>120</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ink Free</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>联系开发者</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setting">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>545</y>
      <width>120</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ink Free</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>设置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="github">
    <property name="geometry">
     <rect>
      <x>429</x>
      <y>545</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ink Free</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>github</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>YEFONTJingHongXingKai</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>获取：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="get_tp">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>YEFONTJingHongXingKai</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>热榜</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>电子课本</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>中考真题</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>高考真题</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="get_grade">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>160</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>YEFONTJingHongXingKai</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>一年级</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>二年级</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>三年级</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>四年级</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>五年级</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>六年级</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>初一</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>初二</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>初三</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>高一</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>高二</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>高三</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="get_subject">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>160</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>YEFONTJingHongXingKai</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>语文</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>数学</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>英语</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>历史</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>地理</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>生物</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>政治</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>物理</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>化学</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="get">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>360</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>YEFONTJingHongXingKai</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>点击获取</string>
    </property>
   </widget>
   <widget class="QListWidget" name="get_list">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>360</width>
      <height>180</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ink Free</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="get_choose">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>360</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>YEFONTJingHongXingKai</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="get_browser">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>YEFONTJingHongXingKai</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>默认浏览器打开</string>
    </property>
   </widget>
   <widget class="QPushButton" name="get_url">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>460</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>YEFONTJingHongXingKai</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>复制URL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="get_phone">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>460</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>YEFONTJingHongXingKai</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>手机扫码下载</string>
    </property>
   </widget>
   <widget class="QPushButton" name="get_browser_pan">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>160</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>YEFONTJingHongXingKai</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>浏览器跳转百度网盘</string>
    </property>
   </widget>
   <widget class="QPushButton" name="get_phone_pan">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>500</y>
      <width>180</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>YEFONTJingHongXingKai</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>手机百度网盘扫码获取</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
